ocrd_network/ocrd_network/processing_server.py

```diff
@@ -106,6 +106,12 @@ def __init__(self, config_path: str, host: str, port: int) -> None:
         # but no internal callback was yet invoked
         self.processing_counter_cache = {}
 
+        # Used for keeping track of locked pages for a workspace
+        # Key: `workspace_id` or `path_to_mets` depending on which is provided
```

Comment:
Is that still accurate? Because you later changed

```
- workspace_key = data.workspace_id if data.workspace_id else data.path_to_mets
+ workspace_key = data.path_to_mets
```

Reply:
Good catch! The key is currently just the `path_to_mets`. I have added a TODO to fix it when refactoring. A more proper resolution of `workspace_id` to `path_to_mets` is required.